.gitignore

```diff
@@ -76,3 +76,8 @@ WexLogFileOutput/*
 #==============================================================================#
 bindings/go/llvm/llvm_config.go
 bindings/go/llvm/workdir
+
+#==============================================================================#
+# Default output path use by Visual Studio cmake project builds.
+#==============================================================================#
+out/
```

Comment:
It's an incidental change, but I prefer to mention those in the description lest it be construed as being related to the main thrust of the change

Reply:
Removed from this change.